### ISSUE TITLE
Please publish debian/ directory for seafile-client package

### ISSUE THREAD

**Issue by andreas-h** [haiwen/seafile-client#786]
If the `debian/` directory containing the utility files needed to build the Debian package of _seafile-client_ were included in this repository, then users could address problems like  #772 without having to constantly bug the maintainers about packaging issues ...

Please put the `debian/` directory into this repository!

**Comment by killing** [comment]
The current packaging strategy for Seafile client is to bundle all three components (seaf-daemon, seafile-applet and ccnet) into one debian package. The debian/ directory is only in the seafile project.

**Comment by andreas-h** [comment]
So the seafile project only contains the client packages? That helps, thanks!

**Comment by killing** [comment]
The debian/ directory in seafile project is only used for client package.

**Comment by AnatomicJC** [comment]
Hi guys,

As I'm using Debian Sid, I have the issue with the libssl package dependancy.
After some search about how to build seafile-client debian package, I found this useful post https://groups.google.com/forum/#!msg/seafile/GS8L9g_yY0I/McOS7l7vBPwJ and I currently running seafile-applet installed from a working debian package.
It is quite easy when you know the procedure. Basically, create a working dir and git clone the 4 projects.
For each project, git checkout the latest stable release (v3.0.7 for libsearpc and v5.0.7 for the others) and build a tar.gz file with:

    ./autogen.sh; ./configure; make; make dist

For seafile-client:

    ./cmake .; make; make dist

About build dependancies, have a look here: http://manual.seafile.com/build_seafile/linux.html

Put each tar.gz file in a separate folder (I choosed /seafile-build/tar_gz), then use the https://github.com/haiwen/seafile/blob/master/scripts/build/build-deb.py to build the debian package:

    python build-deb.py --version=5.0.7 --srcdir=/seafile-build/tar_gz/  --ccnet_version=5.0.7 --seafile_version=5.0.7 --seafile_client_version=5.0.7 --libsearpc_version=3.0.7
